src/Domain/IssueType/index.ts

```diff
@@ -1,14 +1,19 @@
+import { v4 as uuid } from 'uuid';
+
 export interface IIssueType 
 {
-   IssueId:string
-   IssueType:string
+   //key:any;
+   IssueId:string;
+   IssueType:string;
 } 
 
 export class IssueType implements IIssueType
 {
+   //public key:any;
```

Comment:
I think you need to uncomment these right?

Reply:
Yeah, will update that.

Reply:
Resolved in https://github.com/LBHackney-IT/lbh-etra-front-end/pull/14/commits/cd35796cfbf5d352ac16f566ab172f842edf4add.